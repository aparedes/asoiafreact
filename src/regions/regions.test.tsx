export {};

// regions: Set<string>,
// selectedRegion: string,
// setRegion: (region: string) => void,

it('Displays the regions', () => {
  // test('Region snapshot', () => {
    // const r = render(
    //   <Regions
    //     regions={Set(['', 'The North', 'Dorne', 'North of The Wall'])}
    //     selectedRegion={''}
    //     setRegion={jest.fn()}
    //   />
    // );

    expect('toupdate').toMatchSnapshot();
  // });

  //
  // regions.find('The North').onClick()
  // regions = component.toJSON()
  // expect(regions).toMatchSnapshot()
});
