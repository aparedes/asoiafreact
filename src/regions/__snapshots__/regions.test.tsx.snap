// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays the regions 1`] = `"toupdate"`;
